.circleci/config.yml

```diff
@@ -81,31 +133,30 @@ jobs:
 workflows:
   build:
     jobs:
+      - build-and-push-base:
+          name: Building the base image
       - build-and-push-python:
-          name: Python 3.8
-          python-version: "3.8"
-      - build-and-push-python:
-          name: Python 3.9
-          python-version: "3.9"
-      - build-and-push-python:
-          name: Python 3.10
-          python-version: "3.10"
-      - build-and-push-python:
-          name: Python 3.11
-          python-version: "3.11"
-
-      - build-and-push-python-datascience:
-          name: Python Datascience 3.8
-          python-version: "3.8"
+          name: Python << matrix.python-version >>
+          matrix:
+            parameters:
+              python-version:
+                - "3.8.19"
```

Comment:
Can you add a link to a table with the exact versions of Python that can be listed here?